### ISSUE TITLE
Provide a way to change the game's ID

### ISSUE THREAD

**Issue by telanus** [lutris/lutris#290]
This is something I've seen now a few times, 

When one add to the database it seems to lock the name so that Lutris will only show you a banner if the name you enter is the same as that in the DB. 

IE Strider 2 won't be accepted as the DB expects Strider II, even if the page in the DB stated Strider 2 (https://lutris.net/games/strider-ii/) as the name of the game.

is there away around this or not?

**Comment by Xodetaetl** [comment]
If you're talking about getting the banner when you add a game to your client that exists on the server, what you can do is add it with the game's name (slug name or ID, as we call it) that the database uses then change it to whatever you want from the config. The ID will remain the same so it will keep in sync with the game on the server.

But obviously this is something we're going to improve. We'll have something like when you want to add a game, you enter the title, the client will search the server database to find matching titles and let you pick the right one from the results.

**Comment by strycore** [comment]
Ok, first thing, I fixed the inconsistency between the ID and the name, Strider II and Strider 2 are different games (Strider 2 was released in 1999 on Playstation and Arcade). If you're looking for Strider 2, it's not in our DB yet.

Then there is a feature currently missing which is changing the ID of an existing game. This has been planned for a while but we haven't come around it yet. I don't think it's on Github so if you don't mind I'll highjack your issue and make it the official one for this feature :p 

Then there is also the whole setting a custom banner thing which is more complex and will come later.
